Formula/rpm.rb

```diff
@@ -27,54 +12,50 @@ class Rpm < Formula
     sha256 "3a276b6a3f7273f8f88effbb057e8d65fc53fa9e812054951bfdd5ce618f54ac" => :yosemite
   end
 
-  depends_on "rpm2cpio" => :build
+  depends_on :gpg => :run
+  depends_on "pkg-config" => :run
   depends_on "berkeley-db"
   depends_on "libmagic"
-  depends_on "popt"
-  depends_on "libtasn1"
+  depends_on "libarchive"
   depends_on "gettext"
+  depends_on "lua"
+  depends_on "openssl@1.1"
+  depends_on "popt"
   depends_on "xz"
-  depends_on "ossp-uuid"
+  depends_on "zstd"
 
   def install
     # only rpm should go into HOMEBREW_CELLAR, not rpms built
-    inreplace "macros/macros.in", "@prefix@", HOMEBREW_PREFIX
+    inreplace "macros.in", "@prefix@", HOMEBREW_PREFIX
+    inreplace "platform.in", "@prefix@", HOMEBREW_PREFIX
+    # prepare for setting correct gpg binary path in postinstall
+    inreplace "macros.in", "@__GPG@", "/usr/bin/gpg2"
```

Comment:
Is this actually needed? Can't we just replace `@__GPG@` in post_install?

Reply:
The configure build system will replace it with whatever it detects. If `gnupg` is already installed, postinstall will fail (I found out as much). This substitution makes it idempotent.

Reply:
>If gnupg is already installed, postinstall will fail (I found out as much).

I think that's only because you had the `depends_on :gpg` when building. If you remove that, it shouldn't see your installed gnupg.

Reply:
You're right. Fixing...